controllers/cloud.redhat.com/providers/kafka/strimzi.go

```diff
@@ -449,6 +435,7 @@ func (s *strimziProvider) configureKafkaConnectCluster() error {
 		Version:          &version,
 		Config:           &config,
 		Image:            &image,
+		Resources:        &s.Env.Spec.Providers.Kafka.Connect.Resources,
```

Comment:
Should we have defaults for connect too?

Reply:
Will add